test/module/irohad/multi_sig_transactions/transport_test.cpp

```diff
@@ -71,12 +63,18 @@ TEST(TransportTest, SendAndReceive) {
       makePeer(addr + std::to_string(port), "abcdabcdabcdabcdabcdabcdabcdabcd");
   // we want to ensure that server side will call onNewState()
   // with same parameters as on the client side
-  EXPECT_CALL(*notifications, onNewState(_, state))
-      .WillOnce(Invoke([&peer](auto &p, auto) { EXPECT_EQ(*p, *peer); }));
+  EXPECT_CALL(*notifications, onNewState(_, _))
+      .WillOnce(
+          Invoke([&peer, &cv, &state](const auto &p, auto const &target_state) {
+            EXPECT_EQ(*peer, *p);
+
+            EXPECT_EQ(state, target_state);
+            cv.notify_one();
+          }));
 
   transport->sendState(*peer, state);
   std::unique_lock<std::mutex> lock(mtx);
-  cv.wait_for(lock, std::chrono::milliseconds(100));
+  cv.wait_for(lock, std::chrono::milliseconds(5000));
```

Comment:
Maybe add a task to rewrite this with gRPC mocks?

Reply:
Have we created it already?